packages/test-utils/package.json

```diff
@@ -14,14 +14,15 @@
   },
   "dependencies": {
     "@jest/types": "^27.4.2",
-    "@types/jest": "*",
```

Comment:
Most probably `@types/jest` is included here because of `test` and `describe`:

https://github.com/facebook/jest/blob/523fa7ee7666322dfad4833b4aa6243da1a11b1b/packages/test-utils/src/ConditionalTest.ts#L18

https://github.com/facebook/jest/blob/523fa7ee7666322dfad4833b4aa6243da1a11b1b/packages/test-utils/src/ConditionalTest.ts#L38